modules/ocf_mirrors/files/rsyncd.conf

```diff
@@ -57,6 +57,14 @@ dont compress = *.gz *.tgz *.zip *.z *.rpm *.deb *.iso *.bz2 *.tbz
 	comment = kali-images mirror
 	path = /opt/mirrors/ftp/kali-images
 
+[kde]
+	comment = kde mirror
+	path = /opt/mirrors/ftp/kde
+
+[kde-applicationdata]
+	comment = kde-applicationdata mirror
+	path = /opt/mirrors/ftp/kde-applicationdata
```

Comment:
If you combine, you'd want to combine here too, although I don't know if they'd want to have these two separate or not, might be best to ask when you send an email asking to join their mirror list.